<launch>
  <env name="PYTHONPATH" value="$(env PYTHONPATH)" />
  <arg name="nao_ip" default="$(optenv NAO_IP 127.0.0.1)" />
  <arg name="nao_port" default="$(optenv NAO_PORT 9559)" />

  <node pkg="pepper_actuators_py" type="people_follow.py" name="pepper_people_follow"  args="--pip=$(arg nao_ip) --pport=$(arg nao_port) " output="screen"></node>
</launch>
